From 9118e58e6061c40ff79fc0cc660b0d040784d066 Mon Sep 17 00:00:00 2001
From: Richard Laager <rlaager@wiktel.com>
Date: Sun, 13 Jan 2019 10:33:29 -0600
Subject: [PATCH 3/5] Fix the PPS threshold in the PPS docs

The threshold is 0.4 s, not 0.5 s.  The code and the other mentions of
this threshold are consistent.  This brings the one outlier in line.
---
 docs/pps.adoc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/docs/pps.adoc b/docs/pps.adoc
index 8ea845428..bfbac81a3 100644
--- a/docs/pps.adoc
+++ b/docs/pps.adoc
@@ -87,7 +87,7 @@ supporting code automatically compiled. Regardles of mechanism, the PPS signal
 must be present and within nominal jitter and wander tolerances. Additionally,
 the prefer peer must be a truechimer; that is, survive the sanity checks and
 intersection algorithm. Finally, the offset of the system clock relative
-to the prefer peer must be within ±0.5 s. The kernel maintains a
+to the prefer peer must be within ±0.4 s. The kernel maintains a
 watchdog timer for the PPS signal; if the signal has not been heard or
 is out of tolerance for more than some interval, currently two minutes,
 the kernel discipline is disabled and operation continues as if it were
-- 
2.17.1

